CHANGELOG.md

```diff
@@ -11,11 +11,17 @@
  * `Address.toPayable`: added a helper to convert between address types without having to resort to low-level casting. ([#1773](https://github.com/OpenZeppelin/openzeppelin-solidity/pull/1773))
  * Facilities to make metatransaction-enabled contracts through the Gas Station Network. ([#1844](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/1844))
  * `Address.sendValue`: added a replacement to Solidity's `transfer`, removing the fixed gas stipend. ([#1962](https://github.com/OpenZeppelin/openzeppelin-solidity/pull/1962))
+ * `PullPayment.withdrawPaymentsWithGas(address payable payee)`: replacement for the deprecated version (see below), forwarding all available gas. ([#1976](https://github.com/OpenZeppelin/openzeppelin-solidity/pull/1976))
+ * `Escrow.withdrawWithGas(address payable payee)`: replacement for the deprecated version (see below), forwarding all available gas. ([#1976](https://github.com/OpenZeppelin/openzeppelin-solidity/pull/1976))
```

Comment:
I think we can merge these two in a single item.

Reply:
I originally had them together, but it was easy to miss the second one if you were skimming over the list of changes. By each having their own bullet point, both will be noticed even on a cursory reading

Reply:
What about using a nested list?

> * Added alternatives to functions that did not forward all gas:
>   * `PullPayment.withdrawPaymentsWithGas(address payable payee)`
>   * `Escrow.withdrawWithGas(address payable payee)`